_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,169 @@
+- analysis.hbonds.WaterBridgeAnalysis will be moved to analysis.hydrogenbonds.WaterBridgeAnalysis.
+
+
+The following parts of the readers/writers will be removed/changed in version 2.0.0:
+- Writer.write_next_timestep() is deprecated in favor of Writer.write().
+- Passing Timestep objects to Writer.write() is deprecated. In 2.0.0 only Universe or AtomGroup objects will be accepted.
+- The way in which the NCDFWriter handles scale factos will change in version 2.x (see Issue #2327 for more details).
+- When writing PDB files, MDAnalysis will no longer be using the last letter of the SegID to set the chainID in version 2.0.0.
+- The bfactors and tempfactors attributes (set by the PDB and MMTF parsers respectively), will be aliased in version 2.0.0.
+- When parsing TPR files, resids will be indexed from 1 rather than the current default of 0.
+
+The following part of the core and library components will be removed/changed in version 2.0.0:
+- lib.log.echo() is deprecated in favor of the new lig.log.ProgressBar.
+- core.universe.as_Universe() is deprecated.
+
+## Author statistics
+
+Altogether this represents the work of 42 contributors from around the world,
+and featured the work of 25 new contributors:
+
+ - [@bdice](https://github.com/bdice)
+ - [@joaomcteixeira](https://github.com/joaomcteixeira)
+ - [@CCook96](https://github.com/CCook96)
+ - [@Yuan-Yu](https://github.com/Yuan-Yu)
+ - [@HTian1997](https://github.com/HTian1997)
+ - [@Iv-Hristov](https://github.com/Iv-Hristov)
+ - [@hmacdope](https://github.com/hmacdope)
+ - [@AnshulAngaria](https://github.com/AnshulAngaria)
+ - [@ss62171](https://github.com/ss62171)
+ - [@Luthaf](https://github.com/Luthaf)
+ - [@yuxuanzhuang](https://github.com/yuxuanzhuang)
+ - [@abhishandy](https://github.com/abhishandy)
+ - [@mlnance](https://github.com/mlnance)
+ - [@shfrz](https://github.com/shfrz)
+ - [@wvandertoorn](https://github.com/wvandertoorn)
+ - [@cbouy](https://github.com/cbouy)
+ - [@AmeyaHarmalkar](https://github.com/AmeyaHarmalkar)
+ - [@Oscuro-Phoenix](https://github.com/Oscuro-Phoenix)
+ - [@andrrizzi](https://github.com/andrrizzi)
+ - [@WG150](https://github.com/WG150)
+ - [@Marcello-Sega](https://github.com/Marcello-Sega)
+ - [@calcraven](https://github.com/calcraven)
+ - [@ianmkenney](https://github.com/ianmkenney)
+ - [@rcrehuet](https://github.com/rcrehuet)
+ - [@hanatok](https://github.com/hanatok)
+
+## Upgrading to MDAnalysis version 1.1.1
+
+To get all these features run either `conda update -c conda-forge mdanalysis`
+or `pip install --upgrade MDAnalysis`.
+
+## Dependencies
+
+We list below the core dependencies and versions that MDAnalysis has been tested on.
+They are provided as a string for easy use with conda or pip:
+
+ - Python 2.7: biopython==1.76 cython==0.29.15 griddataformats==0.5.0 gsd==1.7.0 matplotlib==2.2.5 mmtf-python==1.1.2 netcdf4==1.3.1 numpy==1.16.5 scipy==1.2.1 tqdm==4.60.0
```

Comment:
@IAlibay is this what you meant by requirements.txt? I'm not totally sure what listing the max packages does without listing the Python version to go with.

Reply:
Sure, depends on everyone else as to whether it's the right place to put this information but it seems reasonable to me.

Reply:
looks good to me, especially for a legacy release